### ISSUE TITLE
Ошибка у Source Viewer в /docs/

### ISSUE THREAD

**Issue by 0x4E69676874466F78** [basisjs/basisjs#186]
https://github.com/basisjs/basisjs/blob/8571903014b207a09d45ad2d1bfba277bf21289b/docs/source_viewer.html#L34
должно быть просто
`        var node = new SourceCode({`
Из-за этого даже на оф сайте не показывает исходник если перейти.

**Comment by lahmatiy** [comment]
Да, вероятно это баг. Будет здорово, если сделаете PR. Если нет, то постараюсь поправить сам

**Comment by 0x4E69676874466F78** [comment]
https://github.com/basisjs/basisjs/pull/191#issue-304058739
